### PR TITLE
Implement multicasting

### PR DESCRIPTION
**Pull request by maarten-ic**
Allow multiple conduits connected to a single output port. The only limitation is that the multiplicity of all connected peers must be the same (to avoid interference with vector port functionality).

### CODE REVIEW

**Review by maarten-ic — COMMENTED** [review]
Rename broadcast to multicast

**Review by LourensVeen — CHANGES_REQUESTED** [review]
Nice job! This is looking good overall, but that test bugs me a bit. Could that be fixed?

File: integration_test/test_multicast_cpp.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    # launch MUSCLE Manager with simulation
+    manager = Manager(config, run_dir)
+    manager.start_instances()
+    success = manager.wait()
+
+    # check that all did not go well
+    assert success
+
+
+if __name__ == "__main__":
```
[the remark this answers is not in the document]
Comment (maarten-ic) [reply integration_test/test_multicast_cpp.py]: Fixed with #134 😄

**Comment by LourensVeen** [comment]
Codacy is still complaining, but then it's always complaining, so I'm going to merge this because it's a great new addition to MUSCLE3. Thanks, great work!

**Pull request merged**